03/app.js

```diff
@@ -0,0 +1,12 @@
+const btnList = document.querySelectorAll("button");
+
+const buttonHandler = function () {
+	this.innerText = "clicked";
+	console.log("clicked");
+
+	this.removeEventListener("click", buttonHandler);
+};
+
+btnList.forEach(function (btn) {
```

Comment:
Tutaj nie musimy sprawdzać bo jeśli nie wyszukaliśmy elementów to mamy pustą tablicę i pętla się nie wykona :)